docs/index.md

```diff
@@ -60,13 +65,22 @@ provider "onepassword" {
 }
 ```
 
+Each of the configuration options can be set using env variables.
+That's more secure way to do that as prevents from accidentally pushing sensitive values to the project repo.
```

Comment:
```suggestion
Environment variables are a more secure way to set configuration options because they prevent accidentally pushing sensitive values to the project repo.
```